src/swh_api_functions.py

```diff
@@ -0,0 +1,29 @@
+#
+# Copyright (c) nexB Inc. and others. All rights reserved.
+# heritedcode is a trademark of nexB Inc.
+# SPDX-License-Identifier: Apache-2.0
+# See http://www.apache.org/licenses/LICENSE-2.0 for the license text.
+# See https://github.com/nexB/heritedcode for support or download.
+# See https://aboutcode.org for more information about nexB OSS projects.
+#
+
+
+import requests
+
+# class to genrate our own custom response
+from custom_api_response import CustomApiResponse
+
+BASE_SWH_API_URL = "https://archive.softwareheritage.org/api/1/"
+
+
+def get_content_information(hash_type, hash):
+
+    """
+    this function returns information like checksums, data_url, file_url, license_url
+    based on hash_type and hash as a json object
+    """
+    api = f"{BASE_SWH_API_URL}content/{hash_type}:{hash}/"
+    response = requests.get(api)
+    response = response.json()
+    custom_response = CustomApiResponse(response)
```

Comment:
I would rather see a model object that contains actual concrete fields than a generic data class like this. The value this adds is what when compared with a plain dictionary?
Think about which value this library brings. I think it can be more than merely making a request call?

Reply:
We want structured data in a class that has concrete named instance attributes and not some generic object that can have any attributes. You want a real schema of sorts. Please look at crafting a class https://docs.python.org/3/tutorial/classes.html